nimble/host/src/ble_gap.c

```diff
@@ -3703,6 +3713,25 @@ ble_gap_periodic_adv_start(uint8_t instance)
     return 0;
 }
 
+#if MYNEWT_VAL(BLE_PERIODIC_ADV_ENH)
+static int
+ble_gap_periodic_adv_update_did(uint8_t instance)
+{
+    static uint8_t buf[sizeof(struct ble_hci_le_set_periodic_adv_data_cp)];
+    struct ble_hci_le_set_periodic_adv_data_cp *cmd = (void *) buf;
+    uint16_t opcode;
+    memset(buf, 0, sizeof(buf));
```

Comment:
empty line between variables declarations and code

Reply:
Done.